modules/Lending.py

```diff
@@ -81,6 +81,8 @@ def init(cfg, api1, log1, data, maxtolend, dry_run1, analysis, notify_conf1):
     min_loan_sizes = Config.get_min_loan_sizes()
     dry_run = dry_run1
     transferable_currencies = Config.get_currencies_list('transferableCurrencies')
+    all_currencies = Config.get_list(exchange, 'all_currencies')
```

Comment:
Use `Config.get_all_currencies()`, which has better situation handling and was specifically made for returning this value.

Reply:
Since I cannot comment directly on `get_all_currencies()`:

I would check for a sentinel value before any currencies, and if it is present, ignore the currency.
For example, with a sentinel character of `#`:
```
def get_all_currencies():
    '''
    Get list of all supported currencies by exchange
    '''
    exchange = get_exchange()
    if config.has_option(exchange, 'all_currencies'):
        cur_list = []
        raw_cur_list = config.get(exchange, 'all_currencies').split(',')
        for raw_cur in raw_cur_list:
            cur = raw_cur.strip(' ').upper()
            if (cur[0] != "#"):
                cur_list.append(cur)
        return cur_list
    elif exchange == 'POLONIEX':
        # default, compatibility to old 'Poloniex only' config
        return ['STR', 'BTC', 'BTS', 'CLAM', 'DOGE', 'DASH', 'LTC', 'MAID', 'XMR', 'XRP', 'ETH', 'FCT']
    else:
        raise Exception('ERROR: List of supported currencies must defined in [' + exchange + '] all_currencies.')
```

This way, we can keep the whitelist while also allowing temporary blacklisting.

Reply:
Ok lets use #. Its not perfect because of:
all_currencies = BTC,BTG,#BTG <- BTG will be used 

but it allows more flexibility then nothing.